feature-libs/order/occ/adapters/occ-order-history-extended.adapter.ts

```diff
@@ -0,0 +1,162 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { forkJoin, Observable, of } from 'rxjs';
+import { map, switchMap } from 'rxjs/operators';
+import {
+  Order,
+  Consignment,
+  OrderHistoryList,
+  ReturnRequestList,
+  OrderHistory,
+  ConsignmentTracking,
+} from '@spartacus/order/root';
+import { OccOrderHistoryAdapter } from './occ-order-history.adapter';
+
+@Injectable()
+export class OccOrderHistoryExtendedAdapter extends OccOrderHistoryAdapter {
+  /**
+   * fills the tracking information for each consignment in the input order details
+   * @param userId user id
+   * @param order order details with consignments
+   * @returns order details along with tracking information for each of it's consignments
+   */
+  fillConsignmentTrackingDetails(
```

Comment:
[MEDIUM]
Can we make this method `protected`? I can see it isn't used anywhere outside this class.

Reply:
done